datasets/curiosity_dialogs/curiosity_dialogs.py

```diff
@@ -0,0 +1,220 @@
+    "test_zero": _URL + "curiosity_dialogs.test_zero.json",
+}
+
+
+class CuriosityDialogsConfig(datasets.BuilderConfig):
+    """BuilderConfig for Curiosity Dialogs dataset"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for Curiosity Dialogs dataset.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(CuriosityDialogsConfig, self).__init__(**kwargs)
+
+
+class CuriosityDialogs(datasets.GeneratorBasedBuilder):
+    """Information Seeking in the Spirit of Learning: a Dataset for Conversational Curiosity"""
+
+    VERSION = datasets.Version("1.1.0")
+
+    BUILDER_CONFIGS = [
+        CuriosityDialogsConfig(
+            name="curiosity_dialogs",
+            version=datasets.Version("1.1.0"),
+            description="Curiosity Dialog: A Dataset for Conversational Curiosity",
+        ),
+    ]
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "messages": datasets.Sequence(
+                        {
+                            "message": datasets.Value("string"),
+                            "liked": datasets.ClassLabel(names=["True", "False"]),
+                            "sender": datasets.ClassLabel(names=["user", "assistant"]),
+                            "facts": datasets.Sequence(
+                                {
+                                    "fid": datasets.Value("int32"),
+                                    "used": datasets.ClassLabel(names=["True", "False"]),
+                                    "source": datasets.ClassLabel(names=["section", "known", "random"]),
+                                }
+                            ),
+                            "message_id": datasets.Value("string"),
+                            "dialog_acts": datasets.Sequence(datasets.Value("string")),
+                        }
+                    ),
+                    "known_entities": datasets.Sequence(datasets.Value("string")),
+                    "focus_entity": datasets.Value("string"),
+                    "dialog_id": datasets.Value("int32"),
+                    "inferred_steps": datasets.ClassLabel(names=["True", "False"]),
+                    "created_time": datasets.Value("int64"),
+                    "aspects": datasets.Sequence(datasets.Value("string")),
+                    "first_aspect": datasets.Value("string"),
+                    "second_aspect": datasets.Value("string"),
+                    "shuffle_facts": datasets.ClassLabel(names=["True", "False"]),
+                    "related_entities": datasets.Sequence(datasets.Value("string")),
+                    "tag": datasets.Value("string"),
+                    "user_id": datasets.Value("int32"),
+                    "assistant_id": datasets.Value("int32"),
+                    "is_annotated": datasets.ClassLabel(names=["True", "False"]),
```

Comment:
usually we use the label at index 0 for False, and the one at index 1 for True

can you change that please ? and for the other boolean fields as well ?

you'll need to regenerate the dataset_infos.json after this change